src/screens/TeamFormation/Tabs/Participants.js

```diff
@@ -1,19 +1,60 @@
 import React,  { Component, } from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
 import ParticipantListItem from '../../../components/ParticipantListItem';
 import ListContainer from '../../../containers/ListContainer.js';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import axios from 'axios';
+import {  Text, View } from 'react-native';
+import styled from 'styled-components';
 
 class Participants extends Component {
     constructor(props) {
       super(props);
+      this.state = {
+        participants: []
+      }
+      this.onParticipantSearch = this.onParticipantSearch.bind(this)
+    }
+
+    getConfig = (token) => {
+      return {
+          headers: {
+              'Authorization': `Bearer ${token}`
+          }
+      }
+    }
+
+    onParticipantSearch = async (name) => {
+      let token
+      try{
+        token = await AsyncStorage.getItem("token")
+      } catch(err) {
+        console.log(err)
+        // TODO: redirect to login
+        return
+      }
+
+      axios.get(`http://real.encs.concordia.ca/profile/api/search/?firstName=${name}`, this.getConfig(token))
+        .then(res => this.setState({participants: res.data}))
+        .catch(err => console.log(err))
     }
   
     render() {
       return (
-        <ListContainer marginBottom={50}>
+        <ListContainer marginBottom={50} onSearch={this.onParticipantSearch}>
            {/* placing these for visualization */}
-          <ParticipantListItem
+           {this.state.participants ? this.state.participants.map(participant => (
+             <ParticipantListItem id={participant.id}
+             participantName={`${participant.first_name} ${participant.last_name}`}
+             // commonClasses can be shown only if we keep state of current user's classes
```

Comment:
Common classes we can show if needed. Will require us to store the currently logged user's classes in a global state and then perform operations on it